### ISSUE TITLE
require-hook: false positive error

### ISSUE THREAD

**Issue by akellbl4** [vitest-dev/eslint-plugin-vitest#788]
I'll work on a fix, just wanted to create a ticket for visibility.

Example
```ts
import { describe, it } from "vitest"
import { fn } from "./fn"

describe(fn, () => {
  describe.for(["", null, undefined, false])("empty children", (input) => {
    it("test", ({ expect }) => {
      expect(fn(input)).toBe(true)
    })
})
```
